### ISSUE TITLE
Outcome for bypass bocks

### ISSUE THREAD

**Issue by detlevhfischer** [w3c/wcag3#92]
I was just going to suggest the outcome that would be picking up the substance of 2.4.1 "Bypass Blocks" as a good one to work through since it is a requirement where in WCAG 2.X quite different methods were deemed sufficient (skiplinks, headings, landmarks or their native equivalents) - but I am not sure I can find anything like that in the current list of WCAG 3 outcomes. Is it subsumed under something else?

I think working through the equivalent of 2.4.1 would be interesting especially because there seems to be a grey area as whether to the bypass methods provided are considered sufficient for meeting 2.4.1 (or its WCAG 3 outcome equivalent), and what to do in cases where several of the methods are used simultaneously but may have some deficiencies (say, one of the less important skiplinks fails to work, or the main nav is labelled but two further nav elements aren't; or there is a flood of landmarks making SR navigation confusing; or section markup is incomplete while nav for principal navigation and main are present; or it is just hierarchical headings and nothing else; or the page is very simple and may not need a bypass function at all - and so on).

It would also be an interesting test case for the use of  "accessibility supported" or whatever it is going to morph into (remember the debate whether browser plugins using heading or section markup for sighed keyboard navigation could be deemed sufficient to meet 2.4.1).
